src/txn/txn_rollback_to_stable.c

```diff
@@ -146,6 +148,31 @@ __rollback_row_add_update(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW *rip,
     return (ret);
 }
 
+/*
+ * __rollback_check_if_txnid_non_committed --
+ *     Check if the transaction id is committed.
```

Comment:
The function description should be modified according to the function name.

Reply:
Fixed